tee-worker/local-setup/worker-log-level-config.toml

```diff
@@ -0,0 +1,37 @@
+[tee-worker]
```

Comment:
maybe easier section name: like `[worker]` and `[cli]`
or `[litentry-worker]` and `[litentry-cli]`